src/styles/blocks/card-rate.scss

```diff
@@ -0,0 +1,19 @@
+.rate-stars {
+  display: flex;
+}
+
+.rate-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(../images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+
+  &:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &:last-child {
+    background-image: url(../images/star.svg);
+  }
```

Comment:
should be adaptive as in [stars](https://github.com/mate-academy/layout_stars) task